dartagnan/src/test/java/com/dat3m/dartagnan/c/C11OrigLocksTest.java

```diff
@@ -0,0 +1,101 @@
+package com.dat3m.dartagnan.c;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.configuration.OptionNames;
+import com.dat3m.dartagnan.utils.Result;
+import com.dat3m.dartagnan.utils.rules.Provider;
+import com.dat3m.dartagnan.utils.rules.Providers;
+import com.dat3m.dartagnan.verification.solving.AssumeSolver;
+import com.dat3m.dartagnan.verification.solving.RefinementSolver;
+import com.dat3m.dartagnan.wmm.Wmm;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static com.dat3m.dartagnan.configuration.Arch.C11;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getTestResourcePath;
+import static com.dat3m.dartagnan.utils.Result.*;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class C11OrigLocksTest extends AbstractCTest {
```

Comment:
Why not t o extend `CLockTest` instead and just override the needed methods? I explicitly wanted to avoid having the test completely decoupled to
1) be sure we have the same expected results for both
2) whenever we add a test, we only need to added to one class

Reply:
Right, that should be possible. Will do

Reply:
Done